SimpleScores/SimpleScores.xcodeproj/project.pbxproj

```diff
@@ -467,7 +449,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"SimpleScores/Preview Content\"";
-				DEVELOPMENT_TEAM = B5C26XE59E;
+				DEVELOPMENT_TEAM = 5SYZQ266RF;
```

Comment:
I think changing the development team may be a change added by mistake.

Reply:
It the changes in the ScoreRow that I only want to change